### PR TITLE
chore(core): Automate Daily Integration Tests

### PR DESCRIPTION
**Pull request by lahirumaramba**
- Automate integration tests to run at 06:00 AM (PT) and 06:00 PM (PT) everyday

### CODE REVIEW

**Comment by lahirumaramba** [comment]
It looks like we need to merge this change to master for the workflow to take effect.
```
Scheduled workflows run on the latest commit on the default or base branch.
```
https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#scheduled-events

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Nightly Builds
+
+on:
+  # Runs every day at 06:00 AM (PT) and 06:00 PM (PT) / 02:00 AM (UTC) and 02:00 PM (UTC)
```
Comment (hiranya911) [inline .github/workflows/nightly.yml:18]: Lets make it 6AM and 8PM so it's well out of work hours for everybody.

File: .github/workflows/nightly.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+    # Attach the packaged artifacts to the workflow output. These can be manually
+    # downloaded for later inspection if necessary.
+    - name: Archive artifacts
+      uses: actions/upload-artifact@v1
+      with:
+        name: dist
+        path: dist
+
+    - name: Verify tarball
```
Comment (hiranya911) [inline .github/workflows/nightly.yml:70]: Not necessary for this IMO. But doesn't hurt to have it.

**Review by hiranya911 — APPROVED** [review]
LGTM with a couple of comments.

File: .github/workflows/nightly.yml (hunk 1 above)
Comment (lahirumaramba) [reply .github/workflows/nightly.yml]: Updated to 6am and 8pm PT. Thanks!

File: .github/workflows/nightly.yml (hunk 2 above)
Comment (lahirumaramba) [reply .github/workflows/nightly.yml]: Removed this verification

**Pull request merged**